libs/@guardian/libs/src/storage/README.md

```diff
@@ -119,6 +119,31 @@ Removes all items from storage.
 storage.local.clear();
 ```
 
+## `keys()`
+
+Returns: `string[] | undefined`
+
+Get all of the keys in storage.
+
+### Notes
+
+This is not a method in the native API, and gets all keys by iterating over the
```

Comment:
i think it's ok not to mention this. we have other methods that aren't (although the idea of matching the API is intriguing... we just don't already)